pkgs/development/python-modules/dkimpy/default.nix

```diff
@@ -26,13 +26,17 @@ buildPythonApplication rec {
   '';
 
   postInstall = ''
-    mkdir -p $out/bin $out/libexec
-    mv $out/bin/*.py $out/libexec
-    makeWrapper "$out/libexec/dkimverify.py" $out/bin/dkimverify
-    makeWrapper "$out/libexec/dkimsign.py" $out/bin/dkimsign
-    makeWrapper "$out/libexec/arcverify.py" $out/bin/arcverify
-    makeWrapper "$out/libexec/arcsign.py" $out/bin/arcsign
-    makeWrapper "$out/libexec/dknewkey.py" $out/bin/dknewkey
+    rm $out/bin/*.pyc
+    pushd $out/bin
+    mv arcsign.py arcsign
+    mv arcverify.py arcverify
+    mv dkimsign.py dkimsign
+    mv dkimverify.py dkimverify
+    mv dknewkey.py dknewkey
+    for f in $out/bin/*; do
+      wrapProgram $f --prefix PYTHONPATH ":" $out/${python.sitePackages}/
+    done
+    popd
```

Comment:
Why were these executables renamed in the first place? Is this done in other distributions too?

Reply:
@dotlambda: The man pages refer to dkimverify etc, so that is the rationale...